contributing/development.md

```diff
@@ -8,7 +8,7 @@ This site is powered by Node.js! :sparkles: :turtle: :rocket: :sparkles:
 
 It runs on macOS, Windows, and Linux environments.
 
-You'll need **Node.js v14** to run the site. If you're using [`nodenv`](https://github.com/nodenv/nodenv), read the [`nodenv` docs](#nodenv) for instructions on switching Node.js versions. If you're not using `nodenv`, the best way to install Node.js is to [download the LTS installer from nodejs.org](https://nodejs.org).
+You'll need Node.js version `12 - 14` to run the site. To install Node.js, [download the "LTS" installer from nodejs.org](https://nodejs.org). If you're using [`nodenv`](https://github.com/nodenv/nodenv), read the [`nodenv` docs](#nodenv) for instructions on switching Node.js versions. 
```

Comment:
consider using non-technical terms to talk about the versions. For example, for people not familiar with the Node versioning scheme, `12 - 14` could look like an actual version number. Whereas what I think you mean is "any stable version from 12 on".

Reply:
I chose the literal `12 - 14` string because it's explicit. I can change that to "Node.js versions 12 to 14`

We don't actually mean "any stable version from 12 on". If anyone tries to run the site on something greater than Node 14, it won't run.